### DISCUSSION TITLE
controlling sunsynk only by mqtt without home assistant

### DISCUSSION THREAD

**Discussion by danielbauer** [kellerza/sunsynk#223] [Q&A]
Hi @all,
maybe I'm to blind to see, but is there a possibility to control (read/write) the sunsynk inverter only by mqtt without home assistant?
I bought a rs485 to ethernet modbus adapter (even MQTT is offered), but I don't know how to proceed.
Any hint will be highly appreciated.
Thanks
Danie

**Comment by kellerza** [comment]
Using this addon you could do that. If you have the addon running you could use MQTT explorer to look at your server

**Reply by ghost** [reply]
Is there documentation on how to do this?

**Reply by kellerza** [reply]
http://mqtt-explorer.com/
